mozdownload/scraper.py

```diff
@@ -695,8 +695,8 @@ def get_build_info(self):
             self.build_index = 0
             self.logger.info('Selected build: build%s' % self.build_number)
         else:
-            self.logger.info('Selected build: build%d' %
-                             (self.build_index + 1))
+            self.logger.info('Selected build: %s' %
```

Comment:
The candidate builds are named "build1", "build2", and so on. We don't want to loose this naming convention here.

Reply:
Sorry unable to understand the concern. Display format is still same, output now looks like this:
```
INFO | Retrieving list of candidate builds from https://archive.mozilla.org/pub/firefox/candidates/53.0-candidates/
INFO | Found 4 builds: build1, build3, build5, build6
INFO | Selected build: build6
INFO | Downloading from: https://archive.mozilla.org/pub/firefox/candidates/53.0-candidates/build6/linux-x86_64/en-US/firefox-53.0.tar.bz2
```
When build number is not passed from cli argument, we pick the last entry from `parser.entries` list. Items in this list are already in required convention.

Or your concern is to keep the log statement in the following format?
`self.logger.info('Selected build: build%s' % (<build-number>))`

Reply:
Oh, you are right. I missed that the parser entries are already in the correct form. So this works fine for me.